src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -153,7 +154,6 @@ bg_color = Color( 0.745098, 0.356863, 0.921569, 1 )
 [sub_resource type="StyleBoxEmpty" id=11]
 
 [sub_resource type="Animation" id=12]
-resource_name = "FlashBar"
```

Comment:
Does this break something? Is this an intended change?

Reply:
Unintended change. I don't know why godot removed that.

Reply:
It does like to break random things every now and then to keep you on your toes.